modules/elastodyn/src/ElastoDyn_Registry.txt

```diff
@@ -672,6 +675,9 @@ typedef	^	ParameterType	ReKi	PtfmMass	-	-	-	"Platform mass"
 typedef	^	ParameterType	ReKi	PtfmPIner	-	-	-	"Platform inertia for pitch tilt rotation about the platform CM."
 typedef	^	ParameterType	ReKi	PtfmRIner	-	-	-	"Platform inertia for roll tilt rotation about the platform CM."
 typedef	^	ParameterType	ReKi	PtfmYIner	-	-	-	"Platform inertia for yaw rotation about the platform CM."
+typedef	^	ParameterType	ReKi	PtfmXYIner	-	-	-	"Platform xy inertia about the platform CM"	"kg m^2"
+typedef	^	ParameterType	ReKi	PtfmYZIner	-	-	-	"Platform yz inertia for pitch tilt rotation about the platform CM"	"kg m^2"
+typedef	^	ParameterType	ReKi	PtfmXZIner	-	-	-	"Platform xz inertia for yaw rotation about the platform CM"	"kg m^2"
```

Comment:
Minor comment: Delete "for yaw rotation".

Reply:
@jjonkman Thanks for reviewing the changes! The registry descriptions for the off-diagonal moment of inertia terms are now corrected.